rpc/js/src/dial.ts

```diff
@@ -21,13 +21,16 @@ import {
   CallUpdateRequest,
   CallUpdateResponse,
   ICECandidate,
-  WebRTCConfig,
   OptionalWebRTCConfigRequest,
   OptionalWebRTCConfigResponse,
+  WebRTCConfig,
 } from './gen/proto/rpc/webrtc/v1/signaling_pb';
 import { SignalingService } from './gen/proto/rpc/webrtc/v1/signaling_pb_service';
 import { addSdpFields, newPeerConnectionForClient } from './peer';
 
+import { atob, btoa } from './polyfills';
```

Comment:
do we need to import these here? it doesn't look like we use until later

Reply:
We can't use the builtin versions of `atob` and `btoa` anymore because those are browser only and won't work in react native (and node in the future)

Reply:
understood, but what I meant with my comment is that these functions are not called in this file, so is it necessary to import them here? are they used under the hood somewhere? it looks like we only call them in `rpc/js/src/peer.ts`

Reply:
We call them in this file too: https://github.com/viamrobotics/goutils/pull/249/files#diff-19b1e810c0b7e0d9e2b5bb4c4393faa56cd8c1d4b144c2605b9f1113a61810f9R534

Reply:
whoops, didn't expand enough... should just looked at the full file